docs/contributors/code/testing-overview.md

```diff
@@ -614,6 +614,38 @@ Code style in PHP is enforced using [PHP_CodeSniffer](https://github.com/squizla
 
 To run unit tests only, without the linter, use `npm run test:unit:php` instead.
 
+### Testing Prefixed Functions
+
+Gutenberg's build system automatically prefixes PHP functions with `gutenberg_` to avoid conflicts with WordPress Core. When writing tests for block functions, you must test the **built (prefixed) versions** of functions, not the source versions.
```

Comment:
Presumably when these tests get backported the prefix needs to be removed?

Reply:
Yeh that's where I'm a bit fuzzy. Where should the tests reside?